qiskit/transpiler/passes/scheduling/alignments/align_measures.py

```diff
@@ -118,7 +120,9 @@ def run(self, dag: DAGCircuit):
         """
         time_unit = self.property_set["time_unit"]
 
-        if not _check_alignment_required(dag, self.alignment, Measure):
+        if not _check_alignment_required(
+            dag, self.alignment, Measure
```

Comment:
Looks like `_check_alignment_required` accepts instance, not type

Reply:
yes... I have the same impression. Maybe can be an independent PR a the removal of the TODO?

Reply:
On the further inspection, it is passed to `dag.op_nodes`, which accepts type. It is in fact wrong annotation for `_check_alignment_required`.
https://github.com/Qiskit/qiskit-terra/pull/8266/files#diff-332f5cf793af062738a6d0b057dd29979f0a6b5200e6369dddb06be072a6545fR233